tests/framework/validators/ValidatorTest.php

```diff
@@ -214,4 +214,11 @@ public function testAddError()
         $errors = $m->getErrors('attr_msg_val');
         $this->assertEquals('attr_msg_val::abc::param_value', $errors[0]);
     }
+
+    public function testGetActiveValidatorsForSafeAttributes()
+    {
+        $model = $this->getTestModel();
+        $validators = $model->getActiveValidators('safe_attr');
+        $this->assertInstanceOf('yii\validators\NumberValidator', array_shift($validators));
```

Comment:
will this test still work when someone adds a new rule to FakedValidationModel? Would be made more stable when assigned to a 'key'